tc-benchmark-tests.sh

```diff
@@ -20,7 +20,17 @@ exec_benchmark()
         --models ${model_file} \
         --wav /tmp/LDC93S1.wav \
         --alphabet /tmp/alphabet.txt \
+        --lm_binary /tmp/lm.binary \
+        --trie /tmp/trie \
         --csv ${csv} \
+        --keep
```

Comment:
I think you will want to remove that `--keep` and make the subsequent call producing a different CSV file.